mythril/analysis/modules/suicide.py

```diff
@@ -54,15 +55,25 @@ def _analyze_state(state, node):
         description += "The remaining Ether is sent to: " + str(to) + "\n"
 
     not_creator_constraints = []
-    if len(state.world_state.transaction_sequence) > 1:
-        creator = state.world_state.transaction_sequence[0].caller
-        for transaction in state.world_state.transaction_sequence[1:]:
-            not_creator_constraints.append(
-                Not(Extract(159, 0, transaction.caller) == Extract(159, 0, creator))
-            )
-            not_creator_constraints.append(
-                Not(Extract(159, 0, transaction.caller) == 0)
-            )
+    if len(state.world_state.transaction_sequence) >= 1:
```

Comment:
this simplifies to True :sweat_smile:

Reply:
I wrote it for safety as I was directly indexing the transaction sequence, but looks like an assert would be better there.